src-docs/src/views/inline_edit/inline_edit_text.tsx

```diff
@@ -42,10 +49,28 @@ export default () => {
 
       <EuiSpacer />
 
+      <EuiCheckbox
+        id="isInvalidCheckbox"
+        label="Toggle Validation"
+        checked={isInvalidChecked}
+        onChange={(e) => toggleIsInvalid(e)}
+      />
+
+      <EuiCheckbox
+        id="isLoadingChecbox"
```

Comment:
v small spelling nit
```suggestion
        id="isLoadingCheckbox"
```